build/build-official-release.sh

```diff
@@ -99,7 +99,7 @@ echo "     with the ${KUBE_RELEASE_VERSION} tag. Mark it as a pre-release."
 echo "  3) Upload the ${KUBE_BUILD_DIR}/kubernetes.tar.gz to GitHub"
 echo "  4) Use this template for the release:"
 echo ""
-echo "## [Documentation](http://releases.k8s.io/${KUBE_RELEASE_VERSION}/docs)"
+echo "## [Documentation](http://releases.k8s.io/${KUBE_RELEASE_VERSION}/docs/README.md)"
 echo "## [Examples](http://releases.k8s.io/${KUBE_RELEASE_VERSION}/examples)"
```

Comment:
I looked at that page, and it doesn't have as strong a TOC as the docs page: https://github.com/GoogleCloudPlatform/kubernetes/tree/master/examples/README.md

It's mostly useless to send users there until that's true.